process/block/preprocess/sovereignChainTransactions.go

```diff
@@ -151,9 +156,68 @@ func (sct *sovereignChainTransactions) shouldContinueProcessingScheduledTx(
 
 	addressHasEnoughBalance := sct.hasAddressEnoughInitialBalance(tx)
 	if !addressHasEnoughBalance {
+		log.Debug("address has not enough initial balance", "sender", sct.pubkeyConverter.Encode(tx.SndAddr))
 		mbInfo.schedulingInfo.numScheduledTxsWithInitialBalanceConsumed++
 		return nil, nil, false
 	}
 
 	return tx, miniBlock, true
 }
+
+func (sct *sovereignChainTransactions) shouldSkipMiniBlock(miniBlock *block.MiniBlock) bool {
+	return !sct.isMiniBlockCorrect(miniBlock.Type)
+}
+
+func (sct *sovereignChainTransactions) isTransactionEligibleForExecution(tx *transaction.Transaction, err error) bool {
+	if isCriticalError(err) {
+		log.Debug("sovereignChainTransactions.isTransactionEligibleForExecution: isCriticalError", "error", err)
```

Comment:
no need for log debug.

Reply:
This should remain, as this could be a critical error coming from the caller side after execution of txProcessor.VerifyTransaction